src/app/nation/nation.component.ts

```diff
@@ -94,4 +96,15 @@ export class NationComponent implements OnInit {
     this.createForm();
   }
 
+  view(url) {
+    if(url){
+      this.jsonp.request('http://' + url + '/configurations/_all_docs?include_docs=true&callback=JSONP_CALLBACK')  
+      .subscribe(res => {
+        this.view_data = res.json().rows.length > 0 ? res.json().rows[0].doc : [];
+      },(error) => console.log(error));
```

Comment:
it needs to hide modal when there is no data which is already done in other PR created by Paul